### ISSUE TITLE
Connection/schema selector toolbar in non DBeaver perspective

### ISSUE THREAD

**Issue by pengster-com** [dbeaver/dbeaver#6488]
#### System information: 
- Windows 10
- STS (SpringToolSuite) v4.22
-  DBeaver 6.1 plugin in 

#### Connection specification:
- Database : Oracle 
- Driver name
- Do you use tunnels or proxies : VPN

#### Describe the problem you're observing:
related to BUG: [DBeaver toolbar in non DBeaver perspective #2840](https://github.com/dbeaver/dbeaver/issues/2840)

DBeaver-related toolbars show up in non-DBeaver perspective
and does not hide/disappear when switching to non-sql file/editor

#### Steps to reproduce:
Opening a file in SQL Editor makes them active 
Switching away from DBeaver perspective do not hide/remove those toolbars
Switching to a Java (non-sql) file makes them inactive (greyed-out), but they remain

A roundabout trick I found that work to hide/remove them for me:
1. Switch away from SQL Editor (to any non-sql file)
2. Switch away from DBeaver perspective
If that still does not work,
3. Make sure your SQL Editor is not active
4. Switch back to DBeaver perspective, and then away from DBeaver perspective again

Not sure why this trick works for me.
I'm not an Eclipse plugin developer but I think the SQL Editor makes them appear and active
but only when switching to/from DBeaver perspective removes them. 
Maybe when switching away from SQL Editor some trigger is needed to hide/remove them.
Evidently, these toolbars can be hidden/removed dynamically. Not because something have been screwed-up in workbench.xmi

**Comment by serge-rider** [comment]
Thanks for the bug report. Indeed, it was a bug in one of recent versions (toolbar visibility was explicitly enabled no matter what editor in what perspective was activated).

You can test the fix in Earlly Access version (https://dbeaver.io/files/ea). In order to install Eclipse plugin use update site https://dbeaver.io/files/ea/update/dbeaver/

**Comment by SwampFalc** [comment]
I was confronted with this issue today after I updated DBeaver.

I was glad to find this early access version, which has solved the biggest issue.

However, one issue remains or has returned (since I had it before and then it somehow went away...) is that whenever I start up Eclipse, your Database Toolbar is visible. It doesn't matter what perspective is open or what files, that toolbar is always there **after the start**.

![image](https://user-images.githubusercontent.com/20422044/63021081-16519180-bea0-11e9-96a6-581c97e7aef3.png)

I put that emphasis there because as soon as I cause a toolbar redraw (even just by changing to a different file in the same perspective) it goes away. So it's not a really big deal, but it is bothersome that I cannot find a way to prevent this from showing up in the first place.

I tried turning on and then off the setting to always show that toolbar, but that did not help.

(If you'd prefer a separate bug report, just let me know).
